cluster/saltbase/salt/monit/init.sls

```diff
@@ -20,6 +20,7 @@ monit:
     - group: root
     - mode: 644
 
+{% if "kubernetes-pool" in grains.get('roles', []) %}
```

Comment:
I search in our codebase, looks like kube-proxy is not installed on any master of all cluster providers.